modules/simulations/airflow-emreks/airflow_emr_eks.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+import random
+from typing import List, cast
+
+# import cdk_nag
+from aws_cdk import Aspects, CfnJson, CfnOutput, Stack, Tags
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_eks as eks
+from aws_cdk import aws_emr as emr
+from aws_cdk import aws_emrcontainers as emrc
+from aws_cdk import aws_iam as iam
+
+# from cdk_nag import NagSuppressions
+from constructs import Construct, IConstruct
+
+"""
+This stack deploys the following:
+- EMR on EKS virtual cluster
+- Airflow with EMR on EKS
```

Comment:
This is deploying "Airflow DAGs using EMR on EKS" not MWAA, right?

Reply:
Clarified in the readme